pkg/config/config.go

```diff
@@ -154,6 +154,7 @@ func getDefaultConfig() Config {
 
 		EnableLocalPortForward: false,
 		EnableVscodeSupport:    false,
+		DisableInputAsCommand:  true,
 	}
 
 }
```

Comment:
The code you provided contains no irregularities or potential issues. Here's an optimization suggestion:

  DisableInputAsCommand should be removed because it is not necessary; this line can be omitted.

Also, there isn't anything more to modify here since the function `getDefaultConfig()` has been optimized properly according to its current form and needs to work seamlessly with other parts of the program.